stock_valuation_account_manual_adjustment/README.rst

```diff
@@ -0,0 +1,86 @@
+.. image:: https://img.shields.io/badge/license-AGPLv3-blue.svg
+   :target: https://www.gnu.org/licenses/agpl.html
+   :alt: License: AGPL-3
+
+=========================================
+Stock Inventory Account Manual Adjustment
+=========================================
+
+This module shows in the product inventory stock value and the accounting
+value, and allows to create accounting adjustment entries to align the
+accounting value to match with the inventory stock value.
+
+Configuration
+=============
+
+Assign the group 'Stock Valuation Account Manual Adjustments' to the users
+that should be allowed to list the valuation discrepancies and to reconcile
+them.
```

Comment:
Maybe some extra comment about the increase and decrease contra-accounts would be helpful.